src/main/java/com/onlinebanking/config/DevConfig.java

```diff
@@ -15,4 +21,13 @@
 @Configuration
 @Profile(ProfileTypeConstants.DEV)
 public class DevConfig {
+
+    /**
+     * A bean has to register the path /h2-console for the h2 database.

```

Comment:
Rewording the comment as "A bean to register...." instead of "A bean has to register...." Makes it better